reload.sls

```diff
@@ -0,0 +1,9 @@
+# Reload postgres config
```

Comment:
What purpose does this state have?

A database user can invoke the SQL command, and all changes should automatically be reloaded anyway when applying a state.

In case you're trying to have a state that can reload configurations on a lot of servers, I'd suggest using: `salt '*' cmd.run 'systemctl reload postgresql'` instead.

Reply:
Whenever you create a user within a salt converge, the config needs to be reloaded. I just wanted to automate this. Your approach would work as well..I have no preference here.

Reply:
This should already be done with the `watch` statement in line 11:

```yaml
    - watch:
      - file: /etc/postgresql/{{ version }}/main/*
```

Reply:
Yes, I also thought this. However in my test runs this wasn't the case.

Reply:
That's very weird, I've been using this in multiple environments and never had this problem. Can you doublecheck? The reload only gets triggert if the config file actually was changed; maybe a change was applied before and then the resulting config didn't change?